src/styles/main.scss

```diff
@@ -18,8 +18,8 @@ body {
 
 .spider {
   position: absolute;
-  top: 0;
-  left: 0;
+  top: 10px;
+  left: 10px;
```

Comment:
You should use only js in this task. Don't change scss files.

Reply:
> You should use only js in this task. Don't change scss files.
Does this have any effect on the result? I just forgot to delete it. You could have approved it )